app/views/ticket_requests/_form.html.haml

```diff
@@ -84,7 +85,7 @@
               -# on create, set user into guests list
               = f.hidden_field :guests, value: @user.name_and_email
 
-            - unless @ticket_request.post_payment?
```

Comment:
I think you can do here:

```ruby
unless post_payment? || event.admin?(current_user)
```

Reply:
This does not work. The unless gets caught in the first eval of post_payment and then stops.